readers-frontend/src/components/utils/articles.ts

```diff
@@ -0,0 +1,57 @@
+import { Prisma } from "@prisma/client";
+import prisma from "../../prisma";
+import { s3URL } from "./s3Url";
```

Comment:
It should be s3URL, case sensitive file systems will break here (mine did)

Reply:
Updated!